nemo_text_processing/text_normalization/en/taggers/tokenize_and_classify.py

```diff
@@ -164,8 +164,9 @@ def __init__(
             v_date_graph = vDateFst(ordinal=v_ordinal_graph, deterministic=deterministic).fst
             time_final = pynini.compose(time_graph, v_time_graph)
             date_final = pynini.compose(date_graph, v_date_graph)
+            ordinal_final = pynini.compose(ordinal_graph, v_ordinal_graph.fst)
             range_graph = RangeFst(
-                time=time_final, date=date_final, cardinal=cardinal, deterministic=deterministic
+                time=time_final, date=date_final, cardinal=cardinal, ordinal=ordinal_final, deterministic=deterministic
```

Comment:
same changes for audio-based normalization are needed. Note, CI doesn't run audio-based pytests.